site/mkdocs.yml

```diff
@@ -78,7 +78,7 @@ markdown_extensions:
       permalink: 🔗
 
 extra:
-  icebergVersion: '1.4.3'
+  icebergVersion: '1.5.0'
```

Comment:
For those who are referring this PR in future. 
we also need to update 'nessieVersion' from `libs.versions.toml`